resources/web/wwi/texture_manager.js

```diff
@@ -0,0 +1,124 @@
+/* global THREE */
+'use strict';
+
+function TextureManager() {
+  if (!TextureManager.instance) {
+    TextureManager.instance = this;
+    this.textures = [];
+    this.loadingTextures = [];
+    this.loadingCubeTextureObjects = [];
+    this.streamingMode = false;
+    this.onTextureLoad = undefined;
```

Comment:
There are several remaining `undefined` (missing the single quotes). We should search and replace all of them.

Reply:
Here is correct to use `undefined` without quotes.
You use quotes only when checking for the type.